README.md

```diff
@@ -130,3 +130,18 @@ Overly complex structures and unnecessary code can be avoided by effective refac
 * Overly-nested and long functions
 * Long parameter lists
 
+
+## Code Reviews
+### [Evaluation Feedback process](https://smartbear.com/learn/code-review/guide-to-code-review-process/)
+* #### Use checklists
+Checklists are the most effective way to eliminate frequently made errors. Code review checklists also provide team members with clear expectations for each type of review and can be helpful to track for reporting and process improvement purposes.
+
+* #### Establish a process for fixing defects found
```

Comment:
Should remove bullet points from headers it looks better that way

Reply:
done